Makefile.config.example

```diff
@@ -46,6 +46,10 @@ BLAS := atlas
 # BLAS_INCLUDE := $(shell brew --prefix openblas)/include
 # BLAS_LIB := $(shell brew --prefix openblas)/lib
 
+# Uncomment and set to 3 if you're using OpenCV 3
+# Comment, or set to 2, if you're using OpenCV 2
+# OPENCV_VERSION := 3
```

Comment:
@lionleaf Let's to move this option below line the IO dependency options to be more clear

https://github.com/lionleaf/caffe/blob/makefile_opencv_imgcodecs/Makefile.config.example#L13